proc-macros/src/render_server.rs

```diff
@@ -79,12 +79,12 @@ impl RpcDescription {
 
 		let mut registered = HashSet::new();
```

Comment:
This tracks method names right? Maybe name it `registered_methods` or `seen_method_names`? And that brings the question, do we need to check aliases too? I guess it's not a big problem if users define duplicate aliases but might be good to keep things tight.
EDIT: nvm, you do check aliases too.

Reply:
Exactly, the code is little !DRY but that's mainly because of that inner closure `check_names`, should ok here

Reply:
I think it makes sense to have this check otherwise it would panic with duplicate names or something, also good to have if you unintentionally register the same alias for two different methods.